agent/src/test/resources/org/openjdk/jmc/agent/converters/test/jfrprobes_template.xml

```diff
@@ -58,99 +51,166 @@
 			</class>
 			<method>
 				<name>printGurkaToString</name>
-				<descriptor>(Lorg/openjdk/jmc/bciagent/test/Gurka;)V</descriptor>
+				<descriptor>(Lorg/openjdk/jmc/agent/test/Gurka;)V</descriptor>
 				<parameters>
 					<parameter index="0">
-						<name>Gurka Attribute</name>
-						<description>The one and only converted Gurk-parameter</description>
+						<name>String Gurka</name>
+						<description>The one and only converted Gurk-parameter
+						</description>
 						<contenttype>None</contenttype>
-						<converter>org.openjdk.jmc.agent.converters.test.GurkConverterString</converter>
+						<converter>org.openjdk.jmc.agent.converters.test.GurkConverterString
+						</converter>
 					</parameter>
 				</parameters>
 			</method>
 		</event>
 		<event id="demo.jfr.convertertest.Int">
-			<name>ConverterEventInt-%TEST_NAME%</name>
-			<description>Defined in the xml file and added by the agent. Should transform the Gurka parameter to an
+			<name>ConverterEventInt</name>
+			<description>Defined in the xml file and added by the agent. Should
+				transform the Gurka parameter to an
 				int.
 			</description>
 			<path>demo/converterevents</path>
 			<stacktrace>true</stacktrace>
-			<class>org.openjdk.jmc.agent.converters.test.InstrumentMeConverter</class>
+			<class>org.openjdk.jmc.agent.converters.test.InstrumentMeConverter
+			</class>
 			<method>
 				<name>printGurkaToInt</name>
-				<descriptor>(Lorg/openjdk/jmc/bciagent/test/Gurka;)V</descriptor>
+				<descriptor>(Lorg/openjdk/jmc/agent/test/Gurka;)V</descriptor>
 				<parameters>
 					<parameter index="0">
-						<name>Gurka Attribute</name>
-						<description>The one and only converted Gurk-parameter</description>
+						<name>Int Gurka</name>
+						<description>The one and only converted Gurk-parameter
+						</description>
 						<contenttype>None</contenttype>
-						<converter>org.openjdk.jmc.agent.converters.test.GurkConverterInt</converter>
+						<converter>org.openjdk.jmc.agent.converters.test.GurkConverterInt
```

Comment:
Thinking more about this, would it actually make more sense to specify a _method_ here? In the end, that's what we want to identify. That'd allow for usage of existing methods named other than `convert()`. WDYT?

Reply:
I was considering it, since it would allow people to have multiple converters to different return types in the same class. It becomes a bit more cumbersome to specify though. Perhaps use convert as the default if only specified as a class, and the method, if specified?

Reply:
Added.